src/components/container/Container.js

```diff
@@ -0,0 +1,32 @@
+import { fetchData } from '../../apiCalls';
+import React, { useEffect, useState } from 'react';
+import Show from '../show_card/ShowCard'
+import './Container.css'
+
+const Container = (props) => {
+  const [ isLoading, setIsLoading ] = useState(true);
+  const [ shows, setShows ] = useState([]);
+
+  //TO DO: includes conditional so that we can use this for our tracks and our years. 
+  useEffect(() => {
+    if (props.type === "shows") {
```

Comment:
I haven't used proptypes yet, seems pretty straightforward. Looking forward to getting the rundown on them.

Reply:
I don't think this is Proptypes. Looks like she is sending a `type` through props in order to conditionally render. But I am really curious if it is going to be beneficial to a reusable container or if it is going to make the code way less clear.

Reply:
Yep, Nina! This is for the conditionally rendering.